scripts/inclusion-bot.yaml

```diff
@@ -317,9 +319,22 @@ triggers:
       - postmortem
     alternatives:
       - project reflection 
-      - end of project review
+      - project or incident review
+      - retrospective
```

Comment:
if one is getting persnickity about agile, a retro is not the same thing as an end-of-project review